raisinbread/test/api/LorisApiAuthenticatedTest.php

```diff
@@ -38,7 +38,7 @@ public function setUp(): void
     {
         parent::setUp();
 
-        $this->_version = 'v0.0.4-dev';
+        $this->_version = 'v0.0.5-dev';
```

Comment:
@driusan I changed v0.0.05-dev to v0.0.4, it works.
https://github.com/aces/Loris/pull/8785